### DISCUSSION TITLE
Use .NET API for React app

### DISCUSSION THREAD

**Discussion by rubenmuchaxo** [community/community#33723] [Other Feature Feedback, Questions, & Ideas] [answered]
Is it hard to create a complex react app from a .NET API?

**Comment by rbento01** [comment, marked as answer]
I have only created 1 app of that sort, and I did find it quite hard.

For context my API was of an anime database. [Link](https://raw.githubusercontent.com/rbento01/API/main/api)
The integration for the Create/Edit/Delete for the .NET database on React was hard.

But you can do it
